lib/support/loadSSLCABundleFile.js

```diff
@@ -2,11 +2,21 @@ const https = require('https');
 const fs = require('fs');
 const log = require('./log');
 
-module.exports = () => {
-  const ca = process.env.SSL_CERT_FILE;
-  if (ca !== undefined) {
-    https.globalAgent.options.ca = [];
-    https.globalAgent.options.ca.push(fs.readFileSync(ca));
-    log.info('Loading cert file.');
+const useSSLCertFile = () => {
+  const useSSL = process.env.FUNCTIONS_USE_SSL_CERT_FILE;
+  return useSSL && useSSL === 'true';
+};
+
+const loadSSLCABundleFile = () => {
+  if (useSSLCertFile()) {
+    const ca = process.env.SSL_CERT_FILE;
+
+    if (ca !== undefined) {
+      https.globalAgent.options.ca = [];
+      https.globalAgent.options.ca.push(fs.readFileSync(ca));
```

Comment:
Since this guy is not on a loop, you can initialize with the element inside of it...

like `https.globalAgent.options.ca = [fs.readFileSync(ca)];`